history/cultures/ancient_group.txt

```diff
@@ -1,63 +1,64 @@
+# Ancient Lore
```

Comment:
You lost unique techs like `innovation_large_soldiers` in this file.

Reply:
wym?  This file has massive soldier tech.

Reply:
![image](https://user-images.githubusercontent.com/74790101/127417237-1eafb860-1933-43dd-a37e-ae9d95eca151.png)